Update 15-workflow-sets.Rmd (#136)
tidymodels/TMwR: 15-workflow-sets.Rmd

## Efficiently screening models {#racing-example}

> One effective method for screening a large set of models efficiently is to use the racing approach described in Section \@ref(racing). With a workflow set, the `workflow_map()` function can be used again. Recall that the `fn` argument is the _function_ to apply to the workflows and, in this case, we can use a value of `"tune_race_anova"`. We also pass an appropriate control object; otherwise the options would be the same as the code in the previous section. 

Reviewer: ```suggestion
One effective method for screening a large set of models efficiently is to use the racing approach described in Section \@ref(racing). With a workflow set, we can use the `workflow_map()` function for this racing approach. Recall that after we pipe in our workflow set, the argument we use is the _function_ to apply to the workflows; in this case, we can use a value of `"tune_race_anova"`. We also pass an appropriate control object; otherwise the options would be the same as the code in the previous section. 
```
